src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const prop = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
```

Comment:
The code correctly splits the input string by semicolons and trims each element. However, consider adding validation to handle cases where the input string might not be formatted as expected, such as missing colons or extra semicolons.